docs/src/routes/docs/[...3]modules/[...1]core/+page.md

```diff
@@ -156,15 +156,15 @@ type Chain = {
   namespace?: 'evm' // string indicating chain namespace. Defaults to 'evm' but will allow other chain namespaces in the future
   // PLEASE NOTE: Some wallets require an rpcUrl, label, and token for actions such as adding a new chain.
   // It is recommended to include rpcUrl, label, and token for full functionality.
-  rpcUrl?: string // Recommended to include. Used for network requests.
+  rpcUrl?: string // Recommended to include. Used for network requests (eg Alchemy or Infura end point).
```

Comment:
Why are other companies specifically called out? Are devs confused by what this value should be?

Reply:
This note was originally made to the core README was made in response to a pull request by Alchemy. It provides clarity that Alchemy's RPC can be used as an alternative to Infura in our examples. Rather than adding Alchemy's RPC to all examples and quickstarts, we wanted users to know that both Alchemy and Infura are valid RPC providers for W3O, ensuring flexibility and provider-agnostic compatibility. I updated the whole `Chain` type here to ensure they were aligned.